docs/rules/no-property-in-node.md

```diff
@@ -25,20 +22,17 @@ module.exports = {
     return {
       'ClassDeclaration, FunctionDeclaration'(node) {
         if ('superClass' in node) {
-          console.log('This is a class declaration:', node);
+          // This is a class declaration
         }
       },
     };
   },
-};
+} satisfies Rule.RuleModule;
```

Comment:
It seems like we're mixing commonjs syntax and TypeScript in this example, which is a bit odd. Can we choose one:
- commonjs with the jsdoc types
- or ESM TypeScript and include any imports for the types

Reply:
Updated.  I changed the examples to `js` and put the type annotation back in.